### PR TITLE
Update dpl-workflow.sh

### PR DESCRIPTION
**Pull request by sawenzel**
TRD is not fundamentally required for the TOF matcher workflow (because we can adjust the track sources)

### CODE REVIEW

**Comment by sawenzel** [comment]
Without this, the dpl workflow for pass4 without TRD would not work because tof-matcher will not be included

File: prodtests/full-system-test/dpl-workflow.sh [hunk 1]
```diff
@@ -410,7 +410,7 @@ has_detectors_reco ITS TPC && has_detector_matching ITSTPC && add_W o2-tpcits-ma
 has_detector_reco FT0 && add_W o2-ft0-reco-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC --pipeline $(get_N ft0-reconstructor FT0 REST)"
 has_detector_reco TRD && add_W o2-trd-tracklet-transformer "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TRD_FILTER_CONFIG --pipeline $(get_N TRDTRACKLETTRANSFORMER TRD REST TRDTRK)"
 has_detectors_reco TRD TPC ITS && [[ ! -z "$TRD_SOURCES" ]] && add_W o2-trd-global-tracking "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TRD_CONFIG $TRD_FILTER_CONFIG --track-sources $TRD_SOURCES" "$TRD_CONFIG_KEY;$ITSMFT_FILES"
-has_detectors_reco TOF TRD TPC ITS && [[ ! -z "$TOF_SOURCES" ]] && add_W o2-tof-matcher-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC --track-sources $TOF_SOURCES --pipeline $(get_N tof-matcher TOF REST TOFMATCH)" "$ITSMFT_FILES"
+has_detectors_reco TOF TPC ITS && [[ ! -z "$TOF_SOURCES" ]] && add_W o2-tof-matcher-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC --track-sources $TOF_SOURCES --pipeline $(get_N tof-matcher TOF REST TOFMATCH)" "$ITSMFT_FILES"
```
Comment (shahor02) [inline prodtests/full-system-test/dpl-workflow.sh]: Actually, can you change these to 2 lines as: 
```
has_detectors_reco TRD TPC && [[ ! -z "$TRD_SOURCES" ]] && add_W o2-trd-global-tracking "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC $TRD_CONFIG $TRD_FILTER_CONFIG --track-sources $TRD_SOURCES" "$TRD_CONFIG_KEY;$ITSMFT_FILES"
has_detectors_reco TOF TPC && [[ ! -z "$TOF_SOURCES" ]] && add_W o2-tof-matcher-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC --track-sources $TOF_SOURCES --pipeline $(get_N tof-matcher TOF REST TOFMATCH)" "$ITSMFT_FILES"
```

Comment (davidrohr) [reply prodtests/full-system-test/dpl-workflow.sh]: makes sense for me in this way

Comment (sawenzel) [reply prodtests/full-system-test/dpl-workflow.sh]: should be done now

**Comment by sawenzel** [comment]
Has been addressed in #8419.

**Pull request closed without merging**